corehq/apps/cleanup/management/commands/couch_integrity.py

```diff
@@ -1,61 +1,165 @@
+            default=0,
+            type="int",
+            help='Define how much doc counts can be off by'
+        ),
+    )
+
     def handle_label(self, *labels, **options):
+        wiggle = options.get('wiggle', 0)
         with open(labels[0]) as f:
             integrity_config = IntegrityConfig.wrap(json.loads(f.read()))
-            integrity_check(integrity_config)
-
-
-def integrity_check(config):
-    for view in config.views:
-        matches = defaultdict(list)
-        for database in config.databases:
-            params = {
-                'reduce': 'false',
-                'limit': 0
-            }
-            resp = requests.get("{database}/{view}".format(database=database.uri, view=view),
-                                params=params,
-                                headers=database.headers)
-
-            content = json.loads(resp.content)
-            matches[content['total_rows']].append(database.uri)
-
-        print_result(matches, view)
-
-
-def print_result(matches, view):
+            integrity_check(integrity_config, wiggle)
+
+
+def integrity_check(config, wiggle=0):
+    for suite in config.suites:
+        for view in suite.views:
+            matches = defaultdict(list)
+            for couch in config.couches:
+                params = {
+                    'reduce': 'false',
+                    'limit': 0
+                }
+                resp = requests.get("{uri}/{database}/{view}".format(uri=couch.uri,
+                                                                    database=suite.database,
+                                                                    view=view),
+                    params=params,
+                    headers=couch.headers)
+
+                content = json.loads(resp.content)
+                try:
+                    total_rows = content['total_rows']
+
+                except KeyError:
+                    print "Problem getting `total_rows`. Is this a valid couch database?  {}"\
+                        .format(suite.database)
+                else:
+                    matched = False
+                    for wiggle_range, couches in matches.items():
+                        if wiggle_range[0] <= total_rows <= wiggle_range[1]:
+                            matches[wiggle_range].append(couch.uri)
+                            matched = True
+
+                    if not matched:
+                        new_wiggle_range = (total_rows - wiggle, total_rows + wiggle)
+                        matches[new_wiggle_range].append(couch.uri)
+
+            print_result(matches, view, suite.database)
+
+
+def print_result(matches, view, database):
     if not matches:
         return
 
-    print "##### {} #####".format(view)
-
     if len(matches) == 1:
-        print u"All is consistent in each database for this view"
+        print u"All is consistent in {} for view {}".format(database, view)
         return
 
     for rows, databases in matches.items():
```

Comment:
Might want to call this
```python
for rows, couches in matches.items():
```
as I think the databases referred to in this block are couch hosts, not dbs with a particular couch host, right?  Seems like the nomenclature is inconsistent, which is a bit confusing.
(this is non-blocking feedback)